Packs/Whois/ReleaseNotes/1_5_5.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Whois
+
+- Updated the default value of Rate Limit Retry Count parameter and rate_limit_retry_count argument of !ip command from 3 to 0. The change should fix performance issues that caused because of ip lookup retry.
```

Comment:
```suggestion
- Updated the default value of Rate Limit Retry Count parameter and rate_limit_retry_count argument of the **ip** command from 3 to 0. The change should fix performance issues that were caused because of ip lookup retry.
```